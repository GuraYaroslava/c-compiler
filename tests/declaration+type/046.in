char* p1 = "123";

char* p2 = "456";

p1 + p2;