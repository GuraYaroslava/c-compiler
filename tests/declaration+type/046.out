[Ln: 5, Col: 5] Error: expression must have integral type
